node/package.json

```
@@ -0,0 +1,16 @@
+{
+    "name": "metabase-nodejs-static-embedding-sample",
+    "homepage": "https://github.com/metabase/metabase-nodejs-static-embedding-sample",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+      "test": "echo \"Error: no test specified\" && exit 1"
```

let's add in a start script

```suggestion
      "start": "node index.js"
```